features/standards/mag/structure/02_headings.feature

```diff
@@ -140,7 +140,19 @@ Feature: Headings
     Then it passes
 
   @html @automated
-  Scenario: Subheading before the first main heading
+  Scenario: Level 2 heading before the first main heading
```

Comment:
I think "Subheading before the main heading" was more in keeping with the guidance